exopy_hqc_legacy/instruments/drivers/visa/yokogawa.py

```diff
@@ -156,10 +156,12 @@ def current(self, set_point):
         NB: does not check the current function.
 
         """
-        self.write(":SOURce:LEVel {}".format(set_point))
-        value = self.query('SOURce:LEVel?')
+        # We now adress the source in mA
+        self.write(":SOURce:LEVel {0:.9f}".format(1e-6*set_point))
+        value = float(self.query('SOURce:LEVel?'))
+        value = 1e6*value
```

Comment:
If this is also related to the uA vs A definition, we should get rid of it. GS200 works fine